tests/components/selection_input.test.ts

```diff
@@ -246,7 +246,6 @@ describe("Selection Input", () => {
   test("input is not filled with highlight when maximum ranges reached", async () => {
     const { model } = await createSelectionInput({ hasSingleRange: true });
     expect(fixture.querySelectorAll("input")).toHaveLength(1);
-    model.dispatch("PREPARE_SELECTION_INPUT_EXPANSION");
```

Comment:
should be `addCellToSelection` below instead of `selectCell`. Otherwise the test doesn't test what it says it tests

Reply:
indeed